images/runtime/node/4.8/Dockerfile.base

```diff
@@ -54,8 +54,10 @@ CMD [ "node" ]
 
 FROM node-original AS main
 
-RUN apt-get update && apt-get install -y \
-  unzip \
+RUN apt-get update \
+  && apt-get upgrade -y \
```

Comment:
Can't we do that in a base image instead? since we're upgrading, all the tools will be in the version-specific layer, which will not be reused and therefore increase the overall disk size usage for App Service

Reply:
yes, good point...we should do it in the `oryx-run-base` instead